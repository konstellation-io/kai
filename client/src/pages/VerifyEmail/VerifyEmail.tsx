import React from 'react';

import StateCircle from '../../components/Shape/StateCircle/StateCircle';
import { STATES } from '../../constants/application';

import styles from './VerifyEmail.module.scss';


function VerifyEmail() {
  return (
    <div className={ styles.bg }>
      <div className={ styles.grid }>
        <div className={ styles.container }>
          <h1>Login link sent</h1>
          <StateCircle animation={STATES.SUCCESS} label="DONE" />;
          <p className={ styles.subtitle }>
            An email with an access link has been sent to your email address.
          </p>
        </div>
      </div>
    </div>
  );
}

export default VerifyEmail;
