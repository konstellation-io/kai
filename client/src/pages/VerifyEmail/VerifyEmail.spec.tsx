import React from 'react';
import { render, cleanup } from '@testing-library/react';
import VerifyEmail from './VerifyEmail';
import '@testing-library/jest-dom/extend-expect'


afterEach(cleanup);

it('renders VerifyEmail without crashing', () => {
  const { container } = render(<VerifyEmail />);
  expect(container).toMatchSnapshot();
});

it('show correct texts', () => {
  const { getByText } = render(<VerifyEmail />);
  expect(getByText('Login link sent')).toBeInTheDocument();
});
