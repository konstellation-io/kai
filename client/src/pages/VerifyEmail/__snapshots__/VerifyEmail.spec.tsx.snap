// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders VerifyEmail without crashing 1`] = `
<div>
  <div
    class="bg"
  >
    <div
      class="grid"
    >
      <div
        class="container"
      >
        <h1>
          Login link sent
        </h1>
        <p
          class="subtitle"
        >
          An email with an access link has been sent to your email address.
        </p>
      </div>
    </div>
  </div>
</div>
`;
